sympy/core/mul.py

```diff
@@ -1113,7 +1113,7 @@ def _matches_noncomm(nodes, targets, repl_dict={}):
     def _matches_add_wildcard(dictionary, state):
         node_ind, target_ind = state
         if node_ind in dictionary:
-            begin, end = dictionary[node_ind]
+            begin = dictionary[node_ind][0]
```

Comment:
I wouldn't change this.

Reply:
But here and above you could use `_` for the unused arg.

Reply:
I prefer `end`: it shows that the dictionary value is a 2-tuple marking the beginning and end which is helpful information when reading the code.